packages/babel-helpers/src/helpers/applyDecs2301.js

```diff
@@ -26,6 +26,12 @@ function createAddInitializerMethod(initializers, decoratorFinishedRef) {
   };
 }
 
+function assertInstanceIfPrivate(has, target) {
+  if (has && !has(target)) {
```

Comment:
`assertInstanceIfPrivate` is a runtime call. Can we call `assertInstanceIfPrivate` only for private element? So we don't have to check `has`.